package.json

```diff
@@ -13,7 +13,7 @@
     "start": "node server",
     "start-dev": "npm run build-client-watch & npm run start-server",
     "start-server": "NODE_ENV='development' nodemon server -e html,js,scss --ignore public",
-    "test": "NODE_ENV='test' DATABASE_URL='postgres://localhost:5432/boilermaker-test' mocha ./**/*.spec.js --compilers js:babel-register"
+    "test": "NODE_ENV='test' DATABASE_URL='postgres://localhost:5432/boilermaker-test' mocha ./server/**/*.spec.js ./server/**/**/*.spec.js ./client/**/*.spec.js --compilers js:babel-register"
```

Comment:
I thought `**` in a shell glob meant recursively to go down any number of directories. Why both `./server/**/*.spec.js` and `./server/**/**/*.spec.js`?

Reply:
That's what I thought too, but unfortunately it doesn't seem to be doing that :/